vividus-engine/src/test/java/org/vividus/log/LoggingTableTransformerMonitorTests.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.log;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.util.List;
+
+import com.github.valfirst.slf4jtest.LoggingEvent;
+import com.github.valfirst.slf4jtest.TestLogger;
+import com.github.valfirst.slf4jtest.TestLoggerFactory;
+
+import org.jbehave.core.configuration.Keywords;
+import org.jbehave.core.model.ExamplesTable.TableProperties;
+import org.jbehave.core.model.TableParsers;
+import org.jbehave.core.steps.ParameterConverters;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class LoggingTableTransformerMonitorTests
+{
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String TRANSFORMER_NAME = "TEST_TRANSFORMER";
+    private static final String APPLYING_LINE = "Applying table transformer: {}";
+    private static final String PROPERTIES_LINE = "Properties: {}";
+    private static final String PARAMETER_LINE = "{}";
+    private static final String BEFORE_APPLYING_FORMAT = APPLYING_LINE + LINE_SEPARATOR
```

Comment:
From my point of view having plain strings would be much simpler to read :)

Reply:
Yes, and originally it was, but the checkstyle said "no".
Perhaps it makes sense then to ignore the rule

Reply:
yep, just suppress it for this class